APISpecification.yaml

```diff
@@ -598,22 +596,15 @@ components:
             that have the same assembly, but different annotations.
       required:
         - genome_id
-        - genome_tag
         - taxonomy_id
         - species_taxonomy_id
-        - common_name
         - scientific_name
         - type
         - is_reference
         - assembly
         - assembly_provider
         - assembly_level
-        - assembly_date
         - annotation_provider
-        - annotation_method
-        - annotation_version
-        - annotation_date
-        - image_url
```

Comment:
Woah! All these fields, apart from image_url, which we agreed to remove, are still required; it's just that their value is allowed to be null.